equinox/nn/_embedding.py

```diff
@@ -161,6 +161,11 @@ def process_heads(
     """
 
     embedding_size: int = field(static=True)
+    theta: float = field(static=True)
+
+    def __init__(self, embedding_size: int, theta: float = 10000.0):
+        self.embedding_size = embedding_size
+        self.theta = theta
```

Comment:
This could be done without the explicit `__init__`:
```python
theta: float = field(static=True, default=10_000.0)
```

Reply:
Ah! I knew `field` must have an option for a default value, I must have overlooked it in the docs!